queries/glimmer/highlights.scm

```diff
@@ -1,113 +1,84 @@
 ; === Tag Names ===
+
 ; Tags that start with a lower case letter are HTML tags
 ; We'll also use this highlighting for named blocks (which start with `:`)
 ((tag_name) @tag
-  (#lua-match? @tag "^:?[%l]"))
-
+  (#match? @tag "^(:)?[a-z]"))
 ; Tags that start with a capital letter are Glimmer components
 ((tag_name) @constructor
-  (#lua-match? @constructor "^%u"))
+  (#match? @constructor "^[A-Z]"))
 
-(attribute_name) @attribute
+(attribute_name) @property
 
 (string_literal) @string
-
 (number_literal) @number
-
 (boolean_literal) @boolean
 
 (concat_statement) @string
 
 ; === Block Statements ===
+
 ; Highlight the brackets
 (block_statement_start) @tag.delimiter
-
 (block_statement_end) @tag.delimiter
 
 ; Highlight `if`/`each`/`let`
-(block_statement_start
-  path: (identifier) @keyword.conditional)
-
-(block_statement_end
-  path: (identifier) @keyword.conditional)
-
-((mustache_statement
-  (identifier) @keyword.conditional)
-  (#lua-match? @keyword.conditional "else"))
+(block_statement_start path: (identifier) @conditional)
+(block_statement_end path: (identifier) @conditional)
+((mustache_statement (identifier) @conditional)
+ (#match? @conditional "else"))
 
 ; == Mustache Statements ===
-; Highlight the whole statement, to color brackets and separators
+
+; Hightlight the whole statement, to color brackets and separators
 (mustache_statement) @tag.delimiter
 
 ; An identifier in a mustache expression is a variable
-((mustache_statement
-  [
-    (path_expression
-      (identifier) @variable)
-    (identifier) @variable
+((mustache_statement [
+  (path_expression (identifier) @variable)
+  (identifier) @variable
   ])
-  (#not-any-of? @variable "yield" "outlet" "this" "else"))
-
+  (#not-match? @variable "yield|outlet|this|else"))
 ; As are arguments in a block statement
-(block_statement_start
-  argument: [
-    (path_expression
-      (identifier) @variable)
-    (identifier) @variable
+(block_statement_start argument: [
+  (path_expression (identifier) @variable)
+  (identifier) @variable
   ])
-
 ; As is an identifier in a block param
-(block_params
-  (identifier) @variable)
-
+(block_params (identifier) @variable)
 ; As are helper arguments
-((helper_invocation
-  argument: [
-    (path_expression
-      (identifier) @variable)
-    (identifier) @variable
+((helper_invocation argument: [
+  (path_expression (identifier) @variable)
+  (identifier) @variable
   ])
-  (#not-eq? @variable "this"))
-
+  (#not-match? @variable "this"))
 ; `this` should be highlighted as a built-in variable
 ((identifier) @variable.builtin
-  (#eq? @variable.builtin "this"))
+  (#match? @variable.builtin "this"))
```

Comment:
That is completely backwards. Prefer eq over any-of over lia-match over match (if absolutely unavoidable).